### PR TITLE
Dockerfile for R container added

### PR DESCRIPTION
**Pull request by Schmedu**
*Issue #, if available:*

*Description of changes:*
Dockerfile for R container added

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jaheba** [comment]
I'd like to argue for creating a new folder where we put our dockerfiles. Maybe `/examples/dockerfiles/` is a good place to start.

That way it's more of a suggestion on how one can create such a container, rather than instructions how that is generally done.

File: Dockerfile.r [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM rpy2/rpy2:2.9.x
+
+ADD . /gluonts
+
+RUN pip3 install /gluonts[shell]
```
Comment (jaheba) [inline Dockerfile.r:5]: We could also just suggest to pip install gluonts from PyPi.

Comment (Schmedu) [reply Dockerfile.r]: You're right. Sounds better

File: examples/dockerfiles/Dockerfile.r [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM rpy2/rpy2:2.9.x
+
+RUN pip3 install --upgrade mxnet==1.6 "gluonts[shell]"
+
+RUN R -e 'install.packages(c("forecast", "nnfor"), repos="https://cloud.r-project.org")'
+
+ENV LC_ALL="C.UTF-8"
+ENV LANG="C.UTF-8"
+
+ENTRYPOINT ["python3", "-m", "gluonts.shell"]
```
Comment (jaheba) [inline examples/dockerfiles/Dockerfile.r:10]:
```suggestion
ENTRYPOINT ["python3", "-m", "gluonts.shell"]

```

**Pull request merged**